Formula/aalib.rb

```diff
@@ -40,6 +40,23 @@ def install
   end
 
   test do
-    system "script", "-q", "/dev/null", bin/"aainfo"
+    output = shell_output("#{bin}/aainfo -width 100 -height 50")
```

Comment:
It looks like the `script -q /dev/null` was originally added due to issues running headless.

Not sure if it will break anything by going back to running `aainfo` directly.
However , there are issues trying to test using TTY-required `script` command on Linux CI.

Reply:
I think this is an acceptable fix.